include/iarc7_motion/PidController.hpp

```diff
@@ -35,7 +37,7 @@ class PidController
             const ros::Time& time,
             double& result,
             double derivative = std::numeric_limits<double>::quiet_NaN(),
-            bool log_debug=false);
+            bool log_debug=true);
```

Comment:
So this makes log_debug true by default unless made false by whatever calls the function?